gettsim/tests/test_data/lohn_st_2.csv

```diff
@@ -0,0 +1,2 @@
+hh_id,tu_id,p_id,jahr,steuerklasse,anz_kindergeld_kinder_tu,bruttolohn_m,wohnort_ost,ges_krankenv_zusatz,alter,hat_kinder,lohn_st,lohn_st_soli
```

Comment:
You specify `ges_krankenv_zusatz` in the data. We don't have a solution for this at the moment. The parameter corresponds to the average additional contribution rate and this is what the test should assume without providing information on that in the data.

Reply:
Is it technically possible to merge lohnsteuer_2 into lohnsteuer while ignoring that `lohn_st_2.csv` and `test_lohnsteuer_2.py` do not pass any test? We created them two weeks ago as dummy files, and they are not even passing the tests on branch lohnsteuer_2 anymore. Otherwise they can just be deleted.

Reply:
No worries about failing tests before merging into `main`!